src/pages/angels.js

```diff
@@ -31,41 +36,47 @@ const AngelsPage = () => {
       <h1>POW!</h1>
       <div>
         <div>
-          <Button
-            onClick={() => placeOrder(BRONZE_SKU)}
-            variant="contained"
-            color="primary"
-          >
-            BRONZE
-          </Button>
+          <AngelCard
+            metalText="Bronze"
+            Pricetext="500 NOK"
```

Comment:
Always use camelCase, so I would change `Pricetext` to `priceText`.

Reply:
👍